CoffeeChat/Controller/EditingViewController.swift

```diff
@@ -82,36 +82,61 @@ class EditingViewController: UIViewController {
     private let tableView = UITableView(frame: .zero, style: .grouped)
 
     // MARK: - Display Settings
-    var showsGroups = true
-    private var showingLess = true
-    private var hideAfter = 3 // Doesn't display more [hideAfter] categories if showingLess is true
+    private var showsGroups = true
+    private var showingLessRows = true
```

Comment:
I think `isShowingLessRows` is more Swifty, but I don't know what the convention is in Pear.

Reply:
Would agree. Most Bools I see are either `isVerb(ing/ed)` or `blankHasOccurred`